openreview/conference/templates/recruit_reviewers_pre_process.py

```diff
@@ -7,9 +7,10 @@ def process(client, note, invitation):
     user = urllib.parse.unquote(note.content['user'])
     hashkey = HMAC.new(HASH_SEED.encode(), digestmod=SHA256).update(user.encode()).hexdigest()
 
+    if hashkey != note.content['key']:
```

Comment:
@celestemartinez I made a small change to throw the key error first, I'm worried that bots can try with different email addresses to guess who was invited.